metrics-monitoring/README.adoc

```diff
@@ -63,44 +63,48 @@ As a developer or administrator, you can view metrics in {product} to visualize
 [role="_abstract"]
 {product} supports the following metrics for Kafka instances and topics. In the {product} web console, a subset of these metrics is displayed in the *Dashboard* page of each Kafka instance.
 
-Broker metrics::
+Cluster metrics::
 +
 --
-* `kafka_broker_quota_softlimitbytes`: Maximum amount of broker storage before producers are throttled
+* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Number of incoming bytes for the cluster in the last five minutes. This metric represents all the data that producers are sending to topics in the cluster. The maximum incoming byte rate is 30 MB per second.
 
-* `kafka_broker_quota_totalstorageusedbytes`: Total storage used by the broker
+* `kafka_namespace:haproxy_server_bytes_out_total:rate5m`: Number of outgoing bytes for the cluster in the last five minutes. This metric represents all the data that producers are receiving from topics in the cluster. The maximum outgoing byte rate is 30 MB per second.
 
-* `kafka_controller_kafkacontroller_global_partition_count`: Number of partitions for each broker in the instance
+* `kafka_namespace:kafka_server_socket_server_metrics_connection_count:sum`: Number of current client connections to the cluster. Kafka clients use persistent connections to interact with brokers in the cluster. For example, a consumer holds a connection to each broker it is receiving data from and a connection to its group coordinator. The maximum number of active connections is 2000.
 
-* `kafka_controller_kafkacontroller_offline_partitions_count`: Number of partitions without an active leader and that are therefore not writable or readable
+* `kafka_namespace:kafka_server_socket_server_metrics_connection_creation_rate:sum`: Number of client connection creations per second for the cluster. Kafka clients use persistent connections to interact with brokers in the cluster. A constant high number of connection creations might indicate a client issue. The maximum connection creation rate is 100 per second.
```

Comment:
What about changing `indicate a client issue` to `indicate a client-side issue`?

Reply:
Just trimming. Saying "client-side issue" is a bit colloquial for official tech docs, when it's meaning an issue with the client, so aka, "client issue" communicates the same thing to me but less casual.

Reply:
That makes sense, thanks